salt/modules/boto_dynamodb.py

```diff
@@ -233,7 +265,7 @@ def describe(table_name, region=None, key=None, keyid=None, profile=None):
     return table.describe()
 
 
-def _extract_index(index_data, global_index=False):
+def extract_index(index_data, global_index=False):
```

Comment:
@cachedout a test is failing here because I don't have the text "Example" in the docstring. It doesn't make sense to call this from the command line. I removed the `_` from the name since it's now called from another Python module.

What should I do here? I am guessing move it to another location.